system/Model.php

```diff
@@ -943,7 +947,7 @@ public function purgeDeleted()
 		}
 
 		return $this->builder()
-						->where($this->deletedField, 1)
+						->where($this->deletedField . ' IS NOT NULL')
```

Comment:
Boy, I feel like we need some `whereNull()`, `whereNotNull()` methods in the future, but I just checked Mysql, Sqlite, and Postgres and they all share the same syntax so I think we're good here for now.

Reply:
Real feels a bit awkward having raw SQL in the query field. I'd be in favor of those functions as well.